### PR TITLE
Failed to close serial port (virtual mode)

### PR DESCRIPTION
**Pull request by haslinghuis**
Fixes: #2377

### CODE REVIEW

File: src/js/serial_backend.js [hunk 1]
```diff
@@ -556,7 +556,7 @@ function onConnect() {
 }
 
 function onClosed(result) {
-    if (result) { // All went as expected
+    if (result || CONFIGURATOR.virtualMode) { // All went as expected
```
Comment (mikeller) [inline src/js/serial_backend.js:559]: Swallowing a falsey result here is a bad idea - we should rather fix this and return a truthy result here if we were in virtual mode.

**Review by mikeller — CHANGES_REQUESTED** [review]

File: src/js/serial_backend.js (hunk 1 above)
Comment (haslinghuis) [reply src/js/serial_backend.js]: Fixed !

File: src/js/VirtualFC.js [hunk 2]
```diff
@@ -19,7 +19,12 @@ const VirtualFC = {
         virtualFC.MIXER_CONFIG.mixer = 3;
 
         virtualFC.MOTOR_DATA = Array.from({length: 8});
-        virtualFC.MOTOR_3D_CONFIG = true;
+        virtualFC.MOTOR_3D_CONFIG = {
+            deadband3d_low: 1406,
```
Comment (mikeller) [inline src/js/VirtualFC.js:23]: Not sure this is needed for this fix?

Comment (haslinghuis) [reply src/js/VirtualFC.js:23]: Yeah maybe make a new issue for it, found another problem (see attached issue)

**Review by mikeller — APPROVED** [review]

**Pull request merged**